release notes/v0.44.0.md

```diff
@@ -0,0 +1,224 @@
+
+    ```javascript
+    const browser = chromium.launch({
+      args: ['no-sandbox'],
+    });
+    ```
+
+- [xk6-browser#844](https://github.com/grafana/xk6-browser/pull/844) Remove the exported `version` param from the root module. Users should from now on reference the k6 version instead of the k6-browser module version.
+- [xk6-browser#838](https://github.com/grafana/xk6-browser/pull/838) Remove first meaningful paint metric. This metric is being deprecated across all the browsers, so we've removed it.
+
+## New features
+
+### xk6-webcrypto is now a built-in module [#2884](https://github.com/grafana/k6/pull/3007)
+
+This release includes [xk6-webcrypto](https://github.com/grafana/xk6-webcrypto) as an experimental module. This is a work in progress implementation of the [WebCrypto API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Crypto_API) specification for k6.
+
+<details>
+<summary> Expand to see an example of the new WebCrypto functionality.</summary>
+
+An example shows encrypting and decrypting of a "Hello, World!" string using AES-CBC algorithm.
+
+```javascript
+import { crypto } from 'k6/experimental/webcrypto';
+
+export default async function () {
+  const key = await crypto.subtle.generateKey(
+    {
+      name: 'AES-CBC',
+      length: 256,
+    },
+    true,
+    ['encrypt', 'decrypt']
+  );
+
+  const encoded = stringToArrayBuffer('Hello, World!');
+  const iv = crypto.getRandomValues(new Uint8Array(16));
+
+  const ciphertext = await crypto.subtle.encrypt(
+    {
+      name: 'AES-CBC',
+      iv: iv,
+    },
+    key,
+    encoded
+  );
+
+  const plaintext = await crypto.subtle.decrypt(
+    {
+      name: 'AES-CBC',
+      iv: iv,
+    },
+    key,
+    ciphertext
+  );
+
+  console.log(
+    'deciphered text == original text: ',
+    arrayBufferToHex(plaintext) === arrayBufferToHex(encoded)
+  );
+}
+
+function arrayBufferToHex(buffer) {
+  return [...new Uint8Array(buffer)].map((x) => x.toString(16).padStart(2, '0')).join('');
+}
+
+function stringToArrayBuffer(str) {
+  var buf = new ArrayBuffer(str.length * 2); // 2 bytes for each char
+  var bufView = new Uint16Array(buf);
+  for (var i = 0, strLen = str.length; i < strLen; i++) {
+    bufView[i] = str.charCodeAt(i);
+  }
+  return buf;
+}
+```
+</details>
+
+The list of supported APIs and algorithms you could find in the project's [README](https://github.com/grafana/xk6-webcrypto/blob/v0.1.0/README.md#current-state)
+
+### Compatibility with z/OS Unix [#2892](https://github.com/grafana/k6/pull/2892)
+
+k6 now supports running on z/OS Unix. This is a new platform that is not officially supported by Golang, but it is based on the same Unix kernel as Linux, so it should work fine. Thanks to @msradam for the contribution!
```

Comment:
We can probably add this to `internal` but as I said above - afero is not compatible, and we do use it still.

I also cannot compile for zos locally so I can't even check if something else is not blocking us on top of that.

Reply:
Fixed in 0c34d12.